src/Raven.Client/Documents/Subscriptions/SubscriptionCriteria.cs

```diff
@@ -5,13 +5,30 @@
 // -----------------------------------------------------------------------
 
 using System;
+using Raven.Client.Documents.Replication;
+using Raven.Client.Documents.Replication.Messages;
 using Sparrow.Json;
 
 namespace Raven.Client.Documents.Subscriptions
 {
+    public class SubscriptionRaftState:IDatabaseTask
+    {
+        public SubscriptionRaftState()
+        {
+
+        }
+        public SubscriptionCriteria Criteria { get; set; }
+        public ChangeVectorEntry[] ChangeVector { get; set; }
+        public long Etag { get; set; }
```

Comment:
Yeah, but that isn't something that the user sees, and it is mainly used for comparisons. This is what the user will see and once created it is fixed, no?